### ISSUE TITLE
Staging Review finding: Alt text for map doesn't describe link destination.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#87425]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Sitewide Facilities
**Product:** VBA Regional Offices
**Feature:** Site Modernization

## Findings details
**VA.gov Experience Standard - issue:** [User is uncertain where a button or link leads.](https://design.va.gov/about/experience-standards/#user-knows-where-a-button-or-link-leads)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the "Location and contact information" section, the image of a map has alt text that just says "Static map."

When an image is the only content of a link, it's alt text needs to communicate both what the image is and where the link is going. As currently coded, only the image is described --- no information on where the link takes you.

## Recommended action

The practical impact here is relatively low, since there's another link in close proximity on the page that also leads to Google Maps for this facility. But I would recommend harmonizing with [ongoing work for the Vet Centers template](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18268#issuecomment-2153102587).

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.1.1 Non-text Content - Level A](https://www.w3.org/TR/WCAG21/#non-text-content)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
